### ISSUE TITLE
ls -la in /proc/1 produces illegal instruction

### ISSUE THREAD

**Issue by itskenny0** [tbodt/ish#274]
```
kyPhoneXS:/proc/1# ls -la                                  
Illegal instruction                                        
kyPhoneXS:/proc/1# dmesg                                   
9 illegal instruction at 0xf7f91d4c: f4 8b 44 24 08 c7 44 2
4                                                          
11 illegal instruction at 0xf7f91d4c: f4 8b 44 24 08 c7 44 
24   
```

**Comment by tbodt** [comment]
dafuq

**Comment by tbodt** [comment]
that's heap corruption!

**Comment by tbodt** [comment]
Fixed in cfc2017
